include/ljmd.h

```diff
@@ -0,0 +1,90 @@
+#ifndef __LJMD__
+#define __LJMD__
+
+
+/* 
+ * simple lennard-jones potential MD code with velocity verlet.
+ * units: Length=Angstrom, Mass=amu; Energy=kcal
+ *
+ * baseline c version.
+ *
+ * latest revision: Stefano Campanella, 02/12/2020
```

Comment:
By experience, adding the latest version is kind of useless. You can forget updating it.